community/libreoffice/APKBUILD

```diff
@@ -198,6 +226,16 @@ prepare() {
 	sed -e 's/@HAVE_GCC_VISIBILITY_BROKEN@/TRUE/g' -i config_host.mk.in
 
 	NOCONFIGURE=1 ./autogen.sh
+
+	mkdir -p "$srcdir"/ext_src
+
+	for t in $_ext_source; do
```

Comment:
Local variables must be declared as `local`. Just use `i` that is already defined in this function.